boltdb/adaptor_test.go

```diff
@@ -0,0 +1,221 @@
+package boltdb_test
+
+import (
+	"context"
+	"errors"
+
+	"github.com/dogmatiq/dogma"
+	"github.com/dogmatiq/enginekit/fixtures"
+	. "github.com/dogmatiq/projectionkit/boltdb"
+	"github.com/dogmatiq/projectionkit/boltdb/internal"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.etcd.io/bbolt"
+)
+
+var _ = Describe("MessageHandler Adaptor", func() {
+	var (
+		mh *internal.MessageHandlerMock
```

Comment:
```suggestion
		handler *internal.MessageHandlerMock
```
I think it's better to use "more descriptive" names for the variables used by all tests.  `db` is fine, but I'd prefer `handler` over `mh` for example.

Reply:
Changed in the latest commits.